x/gov/keeper/keeper.go

```diff
@@ -21,7 +21,7 @@ import (
 type Keeper struct {
 	authKeeper  types.AccountKeeper
 	bankKeeper  types.BankKeeper
-	distrKeeper types.DistributionKeeper
+	DistrKeeper types.DistributionKeeper
```

Comment:
I dont see why we need to make this public

Reply:
Can't we just allow it to be `nil` and only let you set it in app instantiation?

Reply:
ahhh no reason, good call